package.json

```diff
@@ -1,9 +1,14 @@
 {
   "name": "@dimensiondev/webextension-shim",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "repository": "https://github.com/DimensionDev/webextension-polyfill",
   "license": "MIT",
-  "main": "index.js",
+  "main": "./dist/index.js",
+  "module": "./es/index.js",
```

Comment:
This package is not designed to be import-able, it's not a library.

This library doesn't provide any public API (only polyfill for WebExtension API). It generates a single file "webextension-polyfill.js" and a modified version of "typescript.js". The native side should load it directly.

Reply:
OK, remove `main` and `module`?

Reply:
Also the `tsc` building part. Only `rollup` is required.

Reply:
Removed `main` and `module`.